.eslintrc

```diff
@@ -8,29 +8,41 @@
 		"es6": true
 	},
 	"rules": {
-		"strict": 0,
-		"camelcase": 0,
-		"curly": 0,
-		"indent": [2, "tab", { "SwitchCase": 1 }],
-		"eol-last": 1,
-		"no-shadow": 0,
-		"no-redeclare": 2,
-		"no-extra-bind": 1,
-		"no-empty": 0,
-		"no-process-exit": 1,
-		"no-underscore-dangle": 0,
-		"no-use-before-define": 0,
-		"no-undef": 2,
-		"no-unused-vars": 0,
-		"consistent-return": 0,
-		"no-inner-declarations": 1,
-		"no-loop-func": 1,
-		"space-before-function-paren": [2, "never"],
-		"space-before-blocks": [2, "always"],
-		"space-before-keywords": [2, "always"],
-		"no-console": 0,
-		"comma-dangle": 0,
-		"no-unexpected-multiline": 2,
-		"valid-jsdoc": 2
+		"strict": "off",
+		"camelcase": "off",
+		"curly": "off",
+		"indent": ["error", "tab", { "SwitchCase": 1 }],
+		"eol-last": "warn",
+		"no-shadow": "off",
+		"no-redeclare": "error",
+		"no-extra-bind": "warn",
+		"no-empty": "off",
+		"no-process-exit": "warn",
+		"no-underscore-dangle": "off",
+		"no-use-before-define": "off",
+		"no-undef": "error",
+		"no-unused-vars": "off",
+		"consistent-return": "off",
+		"no-inner-declarations": "warn",
+		"no-loop-func": "warn",
+		"space-before-function-paren": ["error", "never"],
+		"space-before-blocks": ["error", "always"],
+		"object-curly-spacing": ["error", "always"],
+		"keyword-spacing": ["error", {
+			"after": false,
+			"overrides": {
+				"try": {"after": true},
+				"else": {"after": true},
+				"throw": {"after": true},
+				"case": {"after": true},
+				"return": {"after": true},
+				"finally": {"after": true},
+				"do": {"after": true}
+			}
+		}],
+		"no-console": "off",
```

Comment:
Later on it'll probably be better to enable this and then specifically disable it on the files / blocks that are allowed to use `console`.